From pavel@ucw.cz Mon Oct 31 15:31:29 2005
Date: Mon, 31 Oct 2005 23:56:26 +0100
From: Pavel Machek <pavel@ucw.cz>
To: Andrew Morton <akpm@osdl.org>, Greg KH <greg@kroah.com>
Subject: USB: kill unneccessary usb-storage blacklist entries
Message-ID: <20051031225626.GA4249@elf.ucw.cz>
Content-Disposition: inline


I actually have this device, and kernel reports blacklist entry is no
longer neccessary.

Signed-off-by: Pavel Machek <pavel@suse.cz>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

 drivers/usb/storage/unusual_devs.h |    5 -----
 1 file changed, 5 deletions(-)

--- gregkh-2.6.orig/drivers/usb/storage/unusual_devs.h	2005-11-02 11:37:03.000000000 -0800
+++ gregkh-2.6/drivers/usb/storage/unusual_devs.h	2005-11-02 12:02:58.000000000 -0800
@@ -710,11 +710,6 @@
                 "DIMAGE E223",
                 US_SC_SCSI, US_PR_DEVICE, NULL, 0 ),
 
-UNUSUAL_DEV(  0x0693, 0x0002, 0x0100, 0x0100, 
-		"Hagiwara",
-		"FlashGate SmartMedia",
-		US_SC_SCSI, US_PR_BULK, NULL, 0 ),
-
 UNUSUAL_DEV(  0x0693, 0x0005, 0x0100, 0x0100,
 		"Hagiwara",
 		"Flashgate",
